src/black/__init__.py

```diff
@@ -127,28 +127,44 @@ def read_pyproject_toml(
 
     if not config:
         return None
+
+    black_config = config.get("config") or (
+        ctx.default_map.get("config") if ctx.default_map else None
+    )
+    if black_config:
+        black_config_path = Path(Path(value).parent, black_config).resolve()
```

Comment:
```suggestion
        black_config_path = (Path(value).parent / black_config).resolve()
```

Reply:
Any reason you prefer that over the current method?

Reply:
Just that I had to have another look to immediately understand what was happening. It's a bit faster (at least for me) to read from left to right: "value's parent / config and resolve that path" rather than "value's parent and config in some call, hold on what was it, oh new Path". But no biggie!